index.js

```diff
@@ -387,6 +388,7 @@ function setup (argv, env) {
     module: __filename,
     deps: {
       foregroundChild: require.resolve('foreground-child'),
+      isWindows: require.resolve('is-windows'),
```

Comment:
Instead of doing this can we just add `isWindows: IS_WINDOWS` to the main settings object?  That way less things get `require`'d by the shim.

Reply:
Good idea, I'll update the shim

Reply:
I updated the PR to send the value directly.